Lib/idlelib/colorizer.py

```diff
@@ -55,26 +55,35 @@ def color_config(text):
 class ColorDelegator(Delegator):
     """Delegator for syntax highlighting (text coloring).
 
-    Class variables:
-        after_id: Identifier for scheduled after event.
+    Instance variables:
+        delegate: Delegator below this one in the stack, meaning the
+                one this one delegates to.
+
+        Used to track state:
+        after_id: Identifier for scheduled after event, which is a
+                timer for colorizing the text.
         allow_colorizing: Boolean toggle for applying colorizing.
         colorizing: Boolean flag when colorizing is in process.
         stop_colorizing: Boolean flag to end an active colorizing
                 process.
         close_when_done: Widget to destroy after colorizing process
                 completes (doesn't seem to be used by IDLE).
```

Comment:
close_when_done is not used by turtledemo.__main__, so we can get rid of it.  I want to do this in its own issue.